doc/system_requirements.md

```diff
@@ -0,0 +1,83 @@
+# System Requirement Specification Exasol Test Container
+
+## Introduction
+
+This maven plugin checks and unifies a project's structure according to the Exasol integration team's repository standards.
+
+## About This Document
+
+### Goal
+
+Goals of this plugin:
+
+* Fasten repository creation
+* Keep repository structure and common files up to date
+
+## Stakeholders
+
+* Exasol Integration Team members
+
+## Features
+
+Features are the highest level requirements in this document that describe the main functionality of ETC.
+
+### Verify repository structure
+`feat~verify-repo-structure~1`
+
+This plugin can verify if the repository structure fits the internal standards.
+
+There are two types of verifications:
+
+* Files that must exist (for example README.md)
+* Files that must exist and have a predefined content (for example GitHub actions)
+
+Needs: req
+
+### Fit repository structure
+`feat~fit-repo-structure~1`
+
+This plugin can create or update the required files or directories.
+
+Needs: req
+
+## Requirements
+
+### Defined repository structure
+`req~repository-structure~1`
+
+This plugin defines the required structure of the repositories.
```

Comment:
The developers define the structure, not the plugin, I think. Rephrase this requirement

Reply:
updated